### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [sivakumarsai/sivakumarsai.github.io#2] [General]
# index

https://sivakumarsai.github.io/

**Comment by sivakumarsai** [comment]
The Laplace transform is

$$\mathcal{L} [f] (s) = \int_0^{\infty}f(t)e^{-st}\ \mathrm{d}t.$$

(test comment)

**Comment by nckpro** [comment]
Do you have games on your phone?

**Comment by hoband** [comment]
https://tenor.com/view/delay-cat-gif-21402724

**Comment by shuerta1** [comment]
wowwwww look at youuuuuu

**Comment by Kensukeken** [comment]
Happy Birthday Metal!

**Comment by KVS02** [comment]
What is your favorite math result?

**Reply by sivakumarsai** [reply]
Not sure. I like a lot of results, so maybe ask me again in a few years. Currently it might be Pontryagin duality.

**Comment by emaileo** [comment]
Hey Sai, 

I attended your talk yesterday at math club (very good talk btw), and as someone who also finds himself often admiring the connection between math and physics, I wanted to ask, what is one of your favorite topics in mathematical physics and why?

**Reply by sivakumarsai** [reply]
Thanks for coming to my talk! I think the theory of distributions is a nice way to make sense of physical phenomena; for example, the delta function is a distribution that finds use in physics and in solving PDEs in general. It is generally used to model phenomena localized at points, which is a decent model. 

The theory of distributions also interacts nicely with the Fourier transform as well. Also look up Green’s functions or find me and ask me about them.

**Comment by maitiennn** [comment]
Hi Sai !!! Post more music content. Come back to UF for candy
